### ISSUE TITLE
console.log not working v1.6

### ISSUE THREAD

**Issue by MadMub** [jasmine/gulp-jasmine-browser#35]
I was wondering how I can perform simple debugging of specs running in phantomjs. `console.log` does not appear to work when using this gulp plugin, do I need to pass any additional config to jasmine and/or phantomjs ? (I've set `console: true`). Or is there another plugin or package I can install for logging to the console while specs are running?

For some context I have been trying to migrate a previous test pipeline that needed browserify, karma, karma-browserify, phantomjs, karma-jasmine,  jasmine to something simpler using just gulp and this package. The previous system's specs supported console-logging so there must be a way to have phantomjs communicate with the console. Any information or help is greatly appreciated.

**Comment by MadMub** [comment]
To follow up, it appears that a change from 1.5 to 1.6 is swallowing console logs, for now I will lock in to v1.5, I will try to help track down the breaking change from 1.5 to 1.6

**Comment by rdy** [comment]
This was changed to optionally print by specifying the onMessage function. Right now the default was to send it nowhere. If you add an onMessage callback to the headless reporter option it should message to where you want.

For example
```
.... .pipe(headless({driver: 'phantomjs' onMessage(message) { console.log(message); }})
```

It wasn't clear if the best thing to do was to send the output to stdout, especially if you are doing test parallelization.

**Comment by rdy** [comment]
fixed in 1.6.1

**Comment by MadMub** [comment]
thank you!
